### ISSUE TITLE
Soundness bug in QF_NRA

### ISSUE THREAD

**Issue by numairmansur** [SRI-CSL/yices2#170]
Hi,
I get the following wrong answer on the following file with the following command:

[88.smt2](https://github.com/SRI-CSL/yices2/files/4244625/88.zip)

`$ ./yices_smt2  --incremental 88.smt2`

```
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
unsat
```

z3 returns:

```
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
```

`$ ./yices_smt2 --version`

```
Yices 2.6.2
Copyright SRI International.
Linked with GMP 6.1.2
Copyright Free Software Foundation, Inc.
Build date: 2020-02-15
Platform: x86_64-pc-linux-gnu (release)
Revision: b5f1d0a72e9e5568938549f830e3b5be2525f053
```

**Comment by dddejan** [comment]
Duplicate of #169

**Comment by numairmansur** [comment]
Interesting.  I thought the bug in #169 is in mcsat ?

**Comment by dddejan** [comment]
> Interesting. I thought the bug in #169 is in mcsat ?

Yes. The QF_NRA logic automatically goes to MCSAT.

**Comment by dddejan** [comment]
Fixed in 84b1f64d151bc18eb570f448fb3e0b4f614d3c46
